src/Microsoft.Health.Fhir.Shared.Web/appsettings.json

```diff
@@ -59,7 +59,7 @@
                 ]
             },
             "Reindex": {
-                "Enabled": false,
+                "Enabled": true,
```

Comment:
>true [](start = 27, length = 4)

Is this change for tests purposes? I'm not sure it's a good idea to enable functionality by default which works only in Cosmos.